src/pages/tailwind/blog.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+
+import { SEO } from 'components';
+import HeadRssLink from 'components/blog/HeadRssLink';
+import ThreeColWithBadges from 'components/tailwind/blog/ThreeColWithBadges';
+import SitewideHeader from 'components/tailwind/SitewideHeader';
+import { Helmet } from 'react-helmet';
+
+const BlogIndex = ({ data }) => {
+  const posts = data.allMarkdownRemark.edges;
+  const siteTitle = data.site.siteMetadata.title;
+
+  return (
+    <>
+      <SEO
+        title={`All blog posts | ${siteTitle}`}
+        description={`
+          Backstage content. Everything from technical how-tos to recaps of community sessions and 
+          general engineering effectiveness content.
+        `}
+      />
+
+      <HeadRssLink />
+      <Helmet>
+        <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
+        <link rel="stylesheet" href="/stylesheets/tailwind.css" />
```

Comment:
These stylesheets will eventually be globally required. During the migration, it is important to not pollute the old pages with new global styles, so I am dynamically loading them on a page-by-page basis.